PocketBeagle/.work-in-progress/Grove/hcsr04/Makefile

```diff
@@ -0,0 +1,17 @@
+#
```

Comment:
This one too. Shouldn't these be moved over to 'seeed-linux-dtverlays' and pre-installed separately in a separate .deb file?